### DISCUSSION TITLE
YetiForce PDF Premium

### DISCUSSION THREAD

**Discussion by ArniSamano** [YetiForceCompany/YetiForceCRM#16220] [How to]
I would like to test PDF Premium, but not sure how to locate it on git.dev version.

**Comment by mariuszkrzaczkowski** [comment]
https://gitdeveloper.yetiforce.com/index.php?module=PDF&parent=Settings&view=Edit
![image](https://user-images.githubusercontent.com/10198654/176302369-48ce3c6a-e321-4244-8da7-3ac335313682.png)

**Reply by ArniSamano** [reply]
Tried generate sales invoice (after changing Generating Engine) but some errors show:

![image](https://user-images.githubusercontent.com/105257433/176307605-ea2ffba4-0038-46f9-b319-c63365b2be41.png)
